lib/human.rb

```diff
@@ -19,6 +19,8 @@ def set_user_token_at(board, coordinates, mark)
         game_array << token
       end
 
-      board.tokens = keys.zip(game_array).to_h
+      game_array.each_with_index do |val, index|
+        board.set_token_at(index + 1, val)
+      end
```

Comment:
- pq se llama set_token_at varias veces / en un for si es un solo movimiento?
- aún usas internals de board

Reply:
Done, se usa set_token varias veces porque no es un solo movimiento, estoy pasando la info de la matriz al hash